### PR TITLE
parallel-benchmark: Upload results

### PR DESCRIPTION
**Pull request by def-**
Worked: https://buildkite.com/materialize/nightly/builds/9483#0191d8ad-4bea-42ba-9aa6-11fccf3dc5a9
### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered. ([trigger-ci for additional test/nightly runs](https://trigger-ci.dev.materialize.com/))
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] If this PR includes major [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note), I have pinged the relevant PM to schedule a changelog post.

### CODE REVIEW

File: misc/python/materialize/test_analytics/setup/tables/12-parallel-benchmark.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+   queries INT,
+   qps DOUBLE,
+   min DOUBLE,
+   max DOUBLE,
+   avg DOUBLE,
+   p50 DOUBLE,
+   p95 DOUBLE,
+   p99 DOUBLE,
+   std DOUBLE,
+   slope DOUBLE
```
Comment (nrainer-materialize) [inline misc/python/materialize/test_analytics/setup/tables/12-parallel-benchmark.sql:28]: They are not nullable, are they?

**Review by nrainer-materialize — APPROVED** [review]
Nice!

**Comment by nrainer-materialize** [comment]
I cannot access the table:

```
select * from parallel_benchmark_result;
Error: permission denied for TABLE "raw.test_analytics.parallel_benchmark_result"
Detail: The 'rainer@materialize.com' role needs SELECT privileges on TABLE "raw.test_analytics.parallel_benchmark_result"
```

**Comment by def-** [comment]
Fixed

**Pull request merged**